ehrapy/data/_datasets.py

```diff
@@ -153,28 +157,74 @@ def diabetes_130(
 
     Examples:
         >>> import ehrapy as ep
-        >>> adata = ep.dt.diabetes_130(encoded=True)
+        >>> adata = ep.dt.diabetes_130_raw(encoded=True)
+
+    References:
+        [1] Beata Strack, Jonathan P. DeShazo, Chris Gennings, Juan L. Olmo, Sebastian Ventura, Krzysztof J. Cios, and John N. Clore, “Impact of HbA1c Measurement on Hospital Readmission Rates: Analysis of 70,000 Clinical Database Patient Records,” BioMed Research International, vol. 2014, Article ID 781670, 11 pages, 2014.
     """
+
     adata = read_csv(
-        dataset_path=f"{ehrapy_settings.datasetdir}/diabetes_130.csv",
-        download_dataset_name="diabetes_130.csv",
-        backup_url="https://figshare.com/ndownloader/files/33950546",
+        dataset_path=f"{ehrapy_settings.datasetdir}/diabetes_130_raw.csv",
+        download_dataset_name="diabetes_130_raw.csv",
+        backup_url="https://figshare.com/ndownloader/files/45110029",
         columns_obs_only=columns_obs_only,
-        index_column="encounter_id",
     )
     if encoded:
         return encode(adata, autodetect=True)
 
     return adata
 
 
+def diabetes_130_fairlearn(
+    encoded: bool = False,
+    columns_obs_only: dict[str, list[str]] | list[str] | None = None,
+) -> AnnData:
+    """Loads the preprocessed diabetes-130 dataset by fairlearn
+
+    This loads the dataset from the `fairlearn.datasets.fetch_diabetes_hospital` function.
```

Comment:
Well, it doesn't really use it but rather the same dataset that this function would download. But it's probably fine :)